models/attachments.js

```diff
@@ -0,0 +1,37 @@
+const { Schema, model } = require('mongoose')
+const { FIELD_CANNOT_BE_EMPTY, FIELD_CANNOT_BE_SHORTER, FIELD_CANNOT_BE_LONGER } = require('~/consts/errors')
+const { ATTACHMENT } = require('~/consts/models')
+
+const attachmentSchema = new Schema(
+  {
+    fileName: {
+      type: String,
+      required: [true, FIELD_CANNOT_BE_EMPTY('fileName')],
```

Comment:
i think format of `fileName` has to be changed to `file name` as in 2 checks below
and the same in `contentType`